components/block/element.js

```diff
@@ -207,25 +207,33 @@ export default class Block extends LitElement {
 
   async fetch() {
     this.loading = true;
-    const resp = await fetch(window.location.href);
-    this.loading = false;
-    return resp.text();
+    try {
+      const resp = await fetch(window.location.href);
+      return resp.text();
+    } finally {
+      this.loading = false;
+    }
   }
 
   async refresh() {
-    let html = await this.fetch();
-
-    const template = document.createElement('template');
-    template.innerHTML = html;
-    const currentBlockHtml = template.content.querySelector(
-      `ngl-block[blockId="${this.blockId}"]`
-    );
-
-    this.innerHTML = currentBlockHtml.innerHTML;
-
-    this.dispatchEvent(
-      new Event('ngl:refresh', {bubbles: true, composed: true})
-    );
+    try {
+      let html = await this.fetch();
+
+      const template = document.createElement('template');
+      template.innerHTML = html;
+      const currentBlockHtml = template.content.querySelector(
+        `ngl-block[blockId="${this.blockId}"]`
+      );
+
+      this.innerHTML = currentBlockHtml.innerHTML;
+
+      this.dispatchEvent(
+        new Event('ngl:refresh', {bubbles: true, composed: true})
+      );
+    } catch (e) {
+      console.error(e);
```

Comment:
Remove console

Reply:
This was added with the intention of allow for easier debugging, because otherwise there would be no record of the error's details. Do you think it is unnecessary? If so, I can remove it.